### PR TITLE
update damonsets to apps/v1 in manifests

### LINKED ISSUE #221: move multus manifest daemonsets to apps/v1
**What happened**:
In kubemacpool some manifests contain daemonset with apiVersion extensions/v1beta1, while it should be apps/v1
files: [Multus.yaml](https://raw.githubusercontent.com/k8snetworkplumbingwg/kubemacpool/master/hack/multus/multus.yaml), [cni-plugin.yaml](https://raw.githubusercontent.com/k8snetworkplumbingwg/kubemacpool/master/hack/cni-plugins/cni-plugins.yaml)
**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:

### PR DESCRIPTION
**Pull request by RamLavi**
Signed-off-by: Ram Lavi <ralavi@redhat.com>

**What this PR does / why we need it**:
Currently kubemacpool contains manifests with daemonset with apiVersion extensions/v1beta1. We want to upgrade them to apps/v1.
fixes #221 
**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: hack/multus/multus.yaml [hunk 1]
```diff
@@ -57,7 +57,7 @@ metadata:
   name: multus
   namespace: kube-system
 ---
-apiVersion: extensions/v1beta1
```
Comment (phoracek) [inline hack/multus/multus.yaml:60]: this should not work. with apps/v1, you have to specify `selector`: https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/#create-a-daemonset

**Comment by RamLavi** [comment]
FYI applying cni-plugins.yaml crashes on `ErrImagePull` but I think it's unrelated to this PR change

**Comment by qinqon** [comment]
/retest

network hiccup
```
tep 1/3 : FROM registry.access.redhat.com/ubi8/ubi-minimal
latest: Pulling from ubi8/ubi-minimal
600f7e2abab1: Pulling fs layer
6cd4d9d86398: Pulling fs layer
error pulling image configuration: received unexpected HTTP status: 503 Service Unavailable
Makefile:94: recipe for target 'container' failed
make: *** [container] Error 1
+ EXIT_VALUE=2

```

**Pull request merged**